<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYZ_AYARLAR_BATCH</class>
 <widget class="QDialog" name="XYZ_AYARLAR_BATCH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mali Yıl Başlangıcı : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mali Yıl Sonu  : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QAdakDate" name="adakDate_mali_yil_baslangic_tarihi">
       <property name="whatsThis">
        <string>Tarih Widgeti</string>
       </property>
       <property name="QAdakDate" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QAdakDate" name="adakDate_mali_yil_bitis_tarihi">
       <property name="whatsThis">
        <string>Tarih Widgeti</string>
       </property>
       <property name="QAdakDate" stdset="0">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_batch_buttons" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAdakDate</class>
   <extends>QWidget</extends>
   <header>QAdakDate.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
